chemprop/v2/cli/train.py

```diff
@@ -247,18 +255,20 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         type=list,
         help="Name of the columns to ignore when :code:`target_columns` is not provided.",
     )
-   
+
     data_args.add_argument(
         "-t",
         "--dataset-type",
-        default="regression", 
-        action=RegistryAction(ReadoutRegistry),  # TODO: is this correct? The choices should be ['regression', 'classification', 'multiclass', 'spectra']
+        default="regression",
+        action=RegistryAction(
+            ReadoutRegistry
+        ),  # TODO: is this correct? The choices should be ['regression', 'classification', 'multiclass', 'spectra']
```

Comment:
I think we might want to change this over to `--task-type` because a `ReadoutFFN` controls the prediction task, which itself implicitly defines the dataset type. That is, there is no internal concept of a dataset type anymore